arex-storage-web-api/src/main/java/com/arextest/storage/service/CacheService.java

```diff
@@ -0,0 +1,33 @@
+package com.arextest.storage.service;
+
+import com.arextest.common.cache.CacheProvider;
+import com.arextest.storage.cache.CacheKeyUtils;
+import javax.annotation.Resource;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+/**
+ * cache related operations
+ * created by xinyuan_wang on 2024/6/25
+ */
+@Service
+@Slf4j
+public class CacheService {
+  private static final long FIVE_MINUTES_EXPIRE = 5 * 60L;
+  private static final byte[] EMPTY_BYTE_ARRAY = new byte[0];
+  @Resource
+  private CacheProvider redisCacheProvider;
+
+  public boolean isInvalidRecord(String recordId) {
+    return redisCacheProvider.get(CacheKeyUtils.buildInvalidRecordKey(recordId)) != null;
```

Comment:
How about using exists instead of get? no need to get results here

Reply:
The arex-common package adds a new exists method